cmd/cluster/core/create.go

```diff
@@ -265,10 +266,25 @@ func GetAPIServerAddressByNode(ctx context.Context) (string, error) {
 func Validate(ctx context.Context, opts *CreateOptions) error {
 	if !opts.Render {
 		client := util.GetClientOrDie()
+		// Validate HostedCluster CustomResource is installed
+		// If the CustomResource doesn't exist, it might indicated `hypershift install` wasn't called
+		hcCRD := &apiextensionsv1.CustomResourceDefinition{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "hostedclusters.hypershift.openshift.io",
+			},
+		}
+		if err := client.Get(ctx, crclient.ObjectKeyFromObject(hcCRD), hcCRD); err != nil {
+			if apierrors.IsNotFound(err) {
+				return fmt.Errorf("CRD %s doesn't exist, make sure to install hypershift resources first (\"hypershift instal\")", crclient.ObjectKeyFromObject(hcCRD))
```

Comment:
typo "instal"

Is this error message giving us something in addition to what we'd get by doing
```
		err := client.Get(ctx, crclient.ObjectKeyFromObject(cluster), cluster)
		if err != nil {
                          return fmt.Errorf(err)
		}
```

Reply:
This is the error returned in case just returning the `Get` operation error:
`ERROR	Failed to create cluster	{"error": "no matches for kind \"HostedCluster\" in version \"hypershift.openshift.io/v1alpha1\""}`
I think that in this case, adding more information (and tip how to fix) to the user might be helpfull (for example in case the user forgot to run `hypershift install`)
This is the error from the new validation I added:
`Failed to create cluster	{"error": "CRD /hostedclusters.hypershift.openshift.io doesn't exist, make sure to install hypershift resources first (\"hypershift install\")"}`